### PR TITLE
Add zip builds to README

### PR DESCRIPTION
**Pull request by hflynn**
Documents the main points of https://github.com/openmicroscopy/ome-documentation/pull/1608 in the README for the docs

### CODE REVIEW

**Comment by hflynn** [comment]
Just click 'view' in the files changed tab to review the rendered doc

File: README.rst [hunk 1]
```diff
@@ -157,13 +157,41 @@ Release number
...
+Zip bundles
+-----------
+
+To build the documentation as a zipped bundle, use::
+
+    ant zip
+
+From the top level directory::
+
+    ant zip -Domero.release="5.3.0-m8"
```
Comment (jburel) [inline README.rst]: maybe don't use internal release

**Comment by hflynn** [comment]
@jburel m8 mentions removed

File: README.rst [hunk 2]
```diff
@@ -157,13 +157,41 @@ Release number
...
+
+will generate the HTML documentation for OMERO and Contributing and create an
+OMERO.doc-5.3.0.zip under omero/_build.
+
+From omero directory::
+
+    ant zip -Domero.release="5.3.0"
+
+will generate the HTML documentation for OMERO and create an
+OMERO.doc-5.3.0.zip under omero/_build.
```
Comment (mtbc) [inline README.rst]: Interesting. Done this way, the `html/_downloads/requirements.txt` doesn't appear in the zip.

File: README.rst [hunk 3]
```diff
@@ -157,13 +157,41 @@ Release number
...
+To build the documentation as a zipped bundle, use::
+
+    ant zip
+
+From the top level directory::
+
+    ant zip -Domero.release="5.3.0"
+
+will generate the HTML documentation for OMERO and Contributing and create an
+OMERO.doc-5.3.0.zip under omero/_build.
```
Comment (mtbc) [inline README.rst]: maybe be clearer that the Contributing docs won't be in this zip?

Comment (mtbc) [reply README.rst]: (nor any other!)

File: README.rst [hunk 4]
```diff
@@ -157,13 +157,41 @@ Release number
...
+    ant zip -Domero.release="5.3.0"
+
+will generate the HTML documentation for OMERO and create an
+OMERO.doc-5.3.0.zip under omero/_build.
+
+From the contributing directory::
+
+    ant zip -Domero.release="5.3.0"
+
+will generate the HTML documentation for contributing and create a
```
Comment (mtbc) [inline README.rst]: "contributing" had an initial capital above

**Comment by hflynn** [comment]
@mtbc should be clearer now

**Review by mtbc — APPROVED** [review]

**Pull request merged**